node/lib/package/LocalPackageSource.ts

```diff
@@ -278,7 +284,7 @@ export class LocalPackageSource extends PackageSource {
                     let schemaModulePath: string = schemaId + "/" + schemaId;
                     schemaModulePaths.push(packageJson.name + "/" + schemaModulePath);
                     schemaInfos.push({
-                        moduleName: schemaModulePath
+                        moduleName: schemaModulePath,
```

Comment:
Yes, tslint checks for it. I usually try to use that pattern anyway, since it makes it easier to add/remove and reorder items.